packages/sales-api-service/package.json

```diff
@@ -46,6 +46,7 @@
     "dot-prop": "6.0.1",
     "hapi-and-healthy": "7.0.7",
     "hapi-swagger": "14.2.5",
+    "http-status-codes": "^2.3.0",
```

Comment:
Don't add in an entire new dependency just for a single line of code - check for the existing pattern, which is to just put in `200`. If this is causing a SonarCloud error, then do what I suggested and declare a `const HTTP_OK = 200` in the authenticate.js file like I mentioned.